src/bson/bson-config.h

```diff
@@ -157,4 +162,14 @@
 # undef BSON_HAVE_DECIMAL128
 #endif
 
+#define BSON_HAVE_RAND_R 1
```

Comment:
Is rand_r available on manylinux1? Windows?

Reply:
I believe it is on Manylinux, but not on windows.